### DISCUSSION TITLE
Github Copilot Chat in Mobile App (iOS): Context of currently opened file could be picked up automatically

### DISCUSSION THREAD

**Discussion by meArchive13** [community/community#123251] [Mobile]
### Select Topic Area

Product Feedback

### Body

I tried Copilot Chat in the mobile app today.

The following scenario: I open a code file in a public repository. Now i want to know something about this file. For example: Can you explain to me what that file does? 

The app always prompts me to tell it an Repository plus File Path. The Chat seems not to know which file is open.
It would be much more convenient if it would derive this information from the opened file automatically.

Is this planned for the future? Or am I missing something or doing something wrong?

**Comment by mattruggio** [comment]
Hey @meneasysoft, thanks for the question.

> Is this planned for the future?

Copilot is a relatively new product and is evolving very fast across all clients including our iOS and Android applications. Context-aware Copilot on mobile features are something currently being investigated. In the future, since you mentioned that the repository is public, this type of functionality may _potentially_ fall into the "Context-aware coding support and explanations" feature under [licensing and plans](https://github.com/features/copilot#pricing). Stay tuned!

**Reply by nighthawk** [reply]
This would be awesome indeed. I expected it to already worked this way as the Copilot button is overlayed in the exact same way that the Comment or Info buttons are. All those other buttons are context-specific but the Copilot button is more a navigation to a separate feature, so I was surprised that the Copilot lacked that context.

**Comment by meArchive13** [comment]
Should work now: https://github.blog/changelog/2024-08-07-whats-new-in-mobile-july-update/

**Comment by Xelliese** [comment]
It’s a hassle having to manually input the file path every time.
